src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -725,7 +725,7 @@ public function writeStream($path, $resource, array $options = [])
      */
     public function url($path)
     {
-        if (isset($this->config['prefix'])) {
+        if (isset($this->config['prefix']) && ! empty($this->config['prefix'])) {
             $path = $this->concatPathToUrl($this->config['prefix'], $path);
```

Comment:
Shouldn't this be handled _inside_ the `concatPathToUrl()` method?

Reply:
good point